tsconfig.json

```diff
@@ -15,18 +15,22 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
What is the reason for this change?

Reply:
Yeah, this was a CRA change with the new react-scripts